internal/quickstart/create_flag.go

```diff
@@ -58,6 +61,14 @@ func (m createFlagModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		default:
 			m.textInput, cmd = m.textInput.Update(msg)
 		}
+	case createdFlagMsg:
+		if !m.success {
```

Comment:
i don't love this name so open to suggestions

Reply:
Does "success" mean that the user created a new flag or tried to create an existing flag so that we would (over)write the flag key with the one from the message? Maybe instead of "success" it could be something like "shouldUpdateFlag"?